SetUp/autoDefine/commandWriters.py

```diff
@@ -193,6 +193,35 @@ def assignSym(botSpecs, entries, defInp):
             else:
                defInp.write('sy '+group+'\n')
 
+#fix() specifies part of geometry to restrain
+#within internal coordinate sub-menu.
+#Only type=tors currently supported.
+#
+#Default: $fix omitted - restraint not used
+#
+#KeyFormat: $fix type=[label] atoms=[label1,label2,label3,label4]
+def fix(botSpecs, entries, defInp):
+   key='$fix'
+
+   fix=getLine(entries, key)
+   fixBot=getLine(botSpecs, key)
+
+   if fixBot:
```

Comment:
Should put a warning here if $fix was already defined, seems likely people would want to tack more frozen coords on.